dec-no-dos/dec.h

```diff
@@ -0,0 +1,129 @@
+/** \file		dec.h
+	\brief		File performing decryption
+	\details	The decryption is based on the fact that the key used in XOR encryption with ECB mode, will be easily spotable over a string of bytes of 0. 
+	\author		Monika Olchowik
+	\date		12.2016
+	\version	1.0
+*/
+
+#ifndef DEC
+#define DEC
+
+#include<algorithm>
```

Comment:
I do not see this file utilizing anything from `<algorithm>`. Try to confine include directives to files where included content is used - with bigger projects it could significantly increase compilation time.